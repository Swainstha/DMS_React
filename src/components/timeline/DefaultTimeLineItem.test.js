import testSnapFunction from '../../util/testSnapFunction';
import DefaulttimeLineItem from './DefaultTimeLineItem'
import React from 'react';
const dummy_data={
   styleName:"style1",
   timeLine:{
    time:"5:45pm",
     title:"title", 
     image:"image1.png", 
     description:"abcde"
   }
}
testSnapFunction("<DefaulttimeLineItem/>", "Snapshot test for DefaulttimeLineItem",<DefaulttimeLineItem {...dummy_data}/> )
