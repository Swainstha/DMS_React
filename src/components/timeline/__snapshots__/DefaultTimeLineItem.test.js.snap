// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DefaulttimeLineItem/> Snapshot test for DefaulttimeLineItem 1`] = `ShallowWrapper {}`;
